### DISCUSSION TITLE
简单根据文档写了个适配 Xray-Core 1.8.1 的 JSON Schema

### DISCUSSION THREAD

**Discussion by ghost** [XTLS/Xray-core#2197] [Show and tell]
以前的 Schema 版本太旧了，最新的也是根据 V2fly 来的，所以自己弄了一个

本来想着自己用的，但是后来想了想可以用小号挂出来

出于安全原因，我不会透露自己是谁，需要可以自取

https://github.com/JHv5B8/Xray-JSON-Schema

完全是手打+照搬文档，可能比自动生成要人性化，但是无法避免出现错误，欢迎纠正

**Comment by yomnxkcs** [comment]
虽然我知道你不介意别人怎么用你的项目，但还是建议加个license。另外建议加个schema-en-us，对有i18n需求的项目友好些。
